src/server/config/auth.js

```diff
@@ -1,11 +1,29 @@
-export function validateAccessTokenFacebook(accessToken) {
-  ///TODO : Validiate accesstoken from facebook API server.
+import request from 'request';
 
-  return false;
+export function crawlByAccessTokenFacebook(accessToken, callback) {
+  // Crawl user data from facebook by access token.
+  request.get('https://graph.facebook.com/me?fields=name,email,locale,timezone,verified&access_token='
+    + accessToken, function (error, response, userBody) {
+    if (!error && response.statusCode == 200) {
+      let result = JSON.parse(userBody);
+      // Crawl user profile_picture from facebook by access token.
+      request.get('https://graph.facebook.com/'
+        + result.id
+        + '/picture?type=large&redirect=0', function (error, response, pictureBody) {
+        if (!error && response.statusCode == 200) {
+          result.profile_picture = JSON.parse(pictureBody).data.url;
+          callback(result);
```

Comment:
I already forgot what callback() method does. Can we please rename this to indicate what this method does?